pkg/coderefs/coderefs.go

```diff
@@ -238,7 +239,7 @@ func generateReferencesFromGrep(flags []string, grepResult [][]string, ctxLines
 		}
 		ref := grepResultLine{Path: path, LineNum: lineNum}
 		if contextContainsFlagKey {
-			ref.FlagKeys = findReferencedFlags(lineText, flags)
+			ref.FlagKeys = findReferencedFlags(lineText, flags, delims)
```

Comment:
ah, so `contextContainsFlagKey` tells us that this line has a flag reference, but in `findReferencedFlags` we'll loop over all flags and return the first flag that matches, regardless of delims.